### PR TITLE
Add examples for PHP and React

### PR DESCRIPTION
**Pull request by NiR-**

### CODE REVIEW

File: examples/api-platform-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# API Platform Demo
+
+##### Prerequisites
+
+To build and run this example you need:
+
+* Docker v18.09+ with experimental features enabled ;
+* docker-compose ;
+* jq and yq (used by the `build` script);
```
Comment (nm2107) [inline examples/api-platform-demo/README.md]: can you make a link to the `yq` tool ? (there are several existing)

File: examples/api-platform-demo/admin/zbuild.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# syntax=akerouanton/zbuilder:200126-30
+kind: nodejs
+version: 12
+#alpine: true
```
Comment (nm2107) [inline examples/api-platform-demo/admin/zbuild.yml]: should this be removed ?

File: examples/laravel-backpack-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+https://github.com/Laravel-Backpack/Demo
+
+##### Prerequisites
+
+To build and run this example you need:
+
+* Docker v18.09+ with experimental features enabled ;
+* docker-compose ;
+* jq and yq (used by the `build` script);
```
Comment (nm2107) [inline examples/laravel-backpack-demo/README.md]: same about `yq` ;)

File: examples/api-platform-demo/admin/zbuild.yml (hunk 2 above)
Comment (NiR-) [reply examples/api-platform-demo/admin/zbuild.yml]: Mh yes and no :smile: I added a `@TODO` comment to remember to uncomment it once Alpine support got merged :wink:

**Pull request merged**